Lombiq.Tests.UI/Models/ElasticsearchRunningContext.cs

```diff
@@ -80,6 +80,7 @@ public Task BeforeTestAsync(UITestContext context) =>
                 // complete, since "GetLastTaskId()" returns only completed tasks.
                 while (lastTaskId > lastFinishedTaskId || lastFinishedTaskId == null)
                 {
+                    cancellation.ThrowIfCancellationRequested();
```

Comment:
Don't you need to stop the `stopWatch` before throwing from cancellation? I'd put the whole `while` block into a `try..finally` and only call `stopWatch.Stop()` in the `finally` once.